### PR TITLE
fix for hostnames

### PR DESCRIPTION
**Pull request by pronix**

### CODE REVIEW

**Comment by lzap** [comment]
Hello,

would you mind reformatting the commit message as:

```
fixes #3067 - domain stripped from hostname for Solaris
```

Thanks. [test]

**Comment by pronix** [comment]
@lzap  fixed

File: app/models/operatingsystems/solaris.rb [hunk 1]
```diff
@@ -96,7 +96,7 @@ def jumpstart_params host, vendor
       :root_server_hostname  => server_name,                            # mediahost
```
Comment (domcleal) [inline app/models/operatingsystems/solaris.rb:96]: Does root_server_hostname also need to be a shortname?  If so, let's move the `.split('.').first` to line 88 and it'll work for both.

Comment (pronix) [inline app/models/operatingsystems/solaris.rb:96]: @domcleal  not require. it is working correct with current server_name

**Comment by domcleal** [comment]
Thanks @pronix, merged as f1ce57f for Foreman 1.4.0.

**Pull request closed without merging**